contracts/rewards/interfaces/IAutoCompoundingRewards.sol

```diff
@@ -78,6 +83,20 @@ interface IAutoCompoundingRewards is IUpgradeable {
      */
     function enableProgram(Token pool, bool status) external;
 
+    /**
+     * @dev sets the number of programs to process the rewards of, when auto-trigerred
+     *
+     * requirements:
+     *
+     * - the caller must be the admin of the contract
+     */
+    function setAutoTriggerCount(uint256 autoTriggerCount) external;
+
+    /**
+     * @dev processes program rewards for several pools
```

Comment:
maybe `processes program rewards based on internal logic, without requiring any input`?

Reply:
I don't see any added value in 'without requiring any input', since it is already obvious from the function prototype.

Reply:
it's the concept and the rationale for its existence

Reply:
Done